bin/disturb.php

```diff
@@ -4,6 +4,7 @@
 use Phalcon\Cli\Console as ConsoleApp;
 use Phalcon\Loader;
 
+define('DISTURB_DEBUG', true);
```

Comment:
it should be better to take the DEBUG value from and env var
to avoid to have to commit this kind of change